Source/WebKit/UIProcess/Cocoa/NavigationState.mm

```diff
@@ -429,7 +429,7 @@ static void interceptMarketplaceKitNavigation(Ref<API::NavigationAction>&& actio
         if (!sourceFrameID || !weakPage)
             return;
 
-        weakPage->addConsoleMessage(*sourceFrameID, MessageSource::Network, MessageLevel::Error, makeString("Can't handle MarketplaceKit link ", url.string().utf8().data(), " due to error: "_s, error));
+        weakPage->addConsoleMessage(*sourceFrameID, MessageSource::Network, MessageLevel::Error, makeString("Can't handle MarketplaceKit link "_s, url.string().utf8().span(), " due to error: "_s, error));
```

Comment:
I don’t think we should convert this to UTF-8 and back to Latin-1/UTF-16, unless there is some benefit to that I am missing. I think we can just write `url.string()` instead of `url.string().utf8().span()`.